### ISSUE TITLE
[Feature Request] Wiimote Control Settings in the Core Options

### ISSUE THREAD

**Issue by Papermanzero** [libretro/dolphin#231]
I thought the last days a little bit about how to integrate dolphin's Wiimote Control and Calibrate Settings in the libretro UI.
Thanks to the new Core Options Submenu, a Menu entry for each Wiimote can be created in the Core Options.
Wiimote Control 1
Wiimote Control 2
Wiimote Control 3
Wiimote Control 4

For each Wiimote Control (1 to 4) the user can then configure following settings:
Wiimote IR: Visible || Hidden (makes the wiimote IR visible or hidden on screen)
Wiimote IR Mode: Relative || Absolut
Vertical Offset: -50 to +50
Total Yaw/Pitch: 0 to 100

In the Control Settings following Options needs then to be added:
Toggle Wiimote IR Visibility (changes from visible to hidden or vice versa)

With that structure, the settings can be easily extended in a sorted manner and most likly all Wiimote Options can be integrated to libretro for each available Wiimote.
With that feature following tickets can be closed:
https://github.com/libretro/dolphin/issues/123
https://github.com/libretro/dolphin/issues/186
https://github.com/libretro/dolphin/issues/212
https://github.com/libretro/dolphin/issues/210

**Comment by bslenul** [comment]
#210 is already fixed:

![image](https://user-images.githubusercontent.com/33353403/133587805-cc2645bb-a091-474e-a9d4-b5ecfe0d9e1d.png)

For #212, the "Wiimote IR" core options will be back with #229 when/if it gets merged.

I really like the idea of submenus for the Wiimotes settings, however the core options for that core currently don't support categories so it would require to convert them to the newest format.

Calibration options would be great for sure, but sounds extremely hard to implement in the core, if even possible.

**Comment by Papermanzero** [comment]
Yes, you are right. I also had a look on the code and it requires a rewrite to support wiimote sub-settings. I think the Toggle Button to Hide/Unhide the Wiimote IR Pointer would be for most of the users good.

**Comment by Papermanzero** [comment]
That’s the code for the Cursor. https://github.com/libretro/dolphin/blob/master/Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.cpp
